Source/UI/Controllers/IssuersScreen/MLCardFormIssuerTableViewCell.swift

```diff
@@ -41,11 +44,35 @@ final class MLCardFormIssuerTableViewCell: UITableViewCell {
 extension MLCardFormIssuerTableViewCell {
     func setupCell(with issuer: MLCardFormIssuer, radioButtonOn: Bool) {
         radioButtonOn ? setupRadioButton(radioButtonOn: true) : setupRadioButton(radioButtonOn: false)
-        if let imageUrl = issuer.imageUrl {
-            issuerImageView.setRemoteImage(imageUrl: imageUrl)
-        }
+        checkHasImage(issuer: issuer)
         accessibilityLabel = issuer.name
     }
+    
+    private func checkHasImage(issuer: MLCardFormIssuer) {
+        guard let imageURL = issuer.imageUrl else {
```

Comment:
You are validating `issuer.imageUrl` but not using it, why?

Reply:
fixed